phsyn-t3/cod-u8776t_phsyn-t3.xml

```diff
@@ -57,881 +57,1053 @@
-            et non econverso.</p>
+          <head xml:id="phsyn-t3-Hd1e385"><supplied>Utrum dictio exclusiva addita toti excludat partem vel
+            econverso</supplied></head>
+          <p xml:id="phsyn-t3-d1e388"><pc type="#pilcrow"/>Septima questio fuit utrum dictio
+          <lb ed="#T"/> exclusiva adiuncta parti integrali excludat totum (vel econverso) et
+            adiunc<lb ed="#T"/>ta minori numero excludat maiorem (vel econverso). Et videtur quod adiuncta
+            <lb ed="#T"/> toti integrali excludat partem, quia sequitur ‘domus valet tantum centum
+            <lb ed="#T"/> libras; non ergo paries’. Ergo addita toti excludit partem (et econverso,
+            <lb ed="#T"/> ut ‘tantum paries est albus; non ergo domus’). <pc type="#pilcrow"/>Sed contra. Propositio exclusiva
+            po<lb ed="#T"/>nit suam preiacentem. Ut ‘tantum domus est colorata; ergo domus est
+            co<lb ed="#T"/>lorata’. Sed si domus est colorata, paries est coloratus, <add place="right-margin">quia posito toto integrali ponitur quelibet pars eius. Ergo a primo: si
+            tantum domus est colorata, paries est coloratus</add>. Ergo dictio exclusiva
+            <lb ed="#T"/> adiuncta parti non excludit totum et non econverso</p>
+
+          <p xml:id="phsyn-t3-d1e391"><pc type="#pilcrow"/> Et dicendum quod tria
+            <lb ed="#T"/> genera accidentium reperiuntur in toto integrali et in parte sua. Quia
+            <lb ed="#T"/> quedam accidentia sunt que conveniunt toti integrali, ut componi ex partibus
+            in<lb ed="#T"/>tegralibus habere partes omnes secundum quantitatem; et propter hoc anima non est
+            <lb ed="#T"/> pars integralis ipsius hominis sed essentialis, et valere centum libras
+            <lb ed="#T"/> convenit ita toti quod non parti. <pc type="#pilcrow"/>Alia autem sunt accidentia que conveniunt tantum parti,
+            <lb ed="#T"/> ut esse <add place="above-line">minus</add> vel sequi ex toto integrali (videlicet partialitas) vel coniungi
+            <lb ed="#T"/> alii quan
+                <subst>
+                  <del rend="erasure"></del>
+                  <add place="above-line">to</add>
+                </subst> ad faciendum maius secundum quantitatem. <pc type="#pilcrow"/>Tertio autem modo sunt quedam
+            <lb ed="#T"/> accidentia que indifferenter conveniunt toti et parti, ut album, nigrum, frigidum,
+            <lb ed="#T"/> calidum, humidum, siccum et consimilia. Et in primo genere accidentis
+            <lb ed="#T"/> dictio exclusiva adiuncta toti integrali excludit partem, ut ‘domus
+            <lb ed="#T"/> valet tantum centum libras; non ergo partees’. <pc type="#pilcrow"/>In aliis duobus generibus
+            <lb ed="#T"/> accidentium dictio exclusiva adiuncta parti semper excludit totum, et n<seg type="correction">
+              <add>on</add>
+              </seg>
+              <lb ed="#T"/> econverso.</p>
```

Comment:
I would change this slightly. the seg element is used to wrap the entire word in which a correction takes place. So I would do: 

`<seg type="correction">n<add>on</add></seg>`

There are few other place where this should be corrected as well. I will try to note them.